pkg/kubectl/cmd/run.go

```diff
@@ -117,9 +117,11 @@ func addRunFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool("expose", false, "If true, a public, external service is created for the container(s) which are run")
 	cmd.Flags().String("service-generator", "service/v2", "The name of the generator to use for creating a service.  Only used if --expose is true")
 	cmd.Flags().String("service-overrides", "", "An inline JSON override for the generated service object. If this is non-empty, it is used to override the generated object. Requires that the object supply a valid apiVersion field.  Only used if --expose is true.")
+	cmd.Flags().Bool("silent", false, "If true, hide the 'waiting for pod to be running' message")
```

Comment:
Help text says this hides the `waiting for pods running` output. Is there other output from the command? It would be more intuitive if `--silent` hid all output except errors.